### ISSUE TITLE
Add insomniac

### ISSUE THREAD

**Issue by neversun** [neversun/bodyweight-timer#7]
Status of b8c3f001c06c146125a1ba51d7a9f1030e495be0:
Project compiles, but does not create an executeable (or can not find it).

Following error is shown
![bildschirmfoto vom 2015-09-13 13 24 25](https://cloud.githubusercontent.com/assets/9157500/9836460/2e7364c0-5a1b-11e5-8de5-45e5c5a1cabe.png)

Compile log: 

```
13:24:13: Running steps for project harbour-bodyweight-timer...
13:24:13: Start SDK: The "MerSDK" virtual machine is already running. Nothing to do.
13:24:13: Starting: "/home/never/.config/SailfishBeta5/mer-sdk-tools/MerSDK/SailfishOS-armv7hl/qmake" /home/never/harbour-bodyweight-timer/harbour-bodyweight-timer.pro -r -spec linux-g++ CONFIG+=debug CONFIG+=declarative_debug CONFIG+=qml_debug

Building target platforms: armv7hl-meego-linux
Building for target armv7hl-meego-linux
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.RMigiJ
Reading /home/mersdk/share/harbour-bodyweight-timer/src/insomniac/insomniac.pro [/home/mersdk/share/build-harbour-bodyweight-timer-MerSDK_SailfishOS_armv7hl-Debug/src/insomniac]
13:24:15: The process "/home/never/.config/SailfishBeta5/mer-sdk-tools/MerSDK/SailfishOS-armv7hl/qmake" exited normally.
13:24:15: Starting: "/home/never/.config/SailfishBeta5/mer-sdk-tools/MerSDK/SailfishOS-armv7hl/make" 
Building target platforms: armv7hl-meego-linux
Building for target armv7hl-meego-linux
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.AzRND5
cd src/insomniac/ && ( test -e Makefile || /usr/lib/qt5/bin/qmake /home/mersdk/share/harbour-bodyweight-timer/src/insomniac/insomniac.pro QMAKE_CFLAGS_RELEASE=-O2\ -g\ -pipe\ -Wall\ -Wp,-D_FORTIFY_SOURCE=2\ -fexceptions\ -fstack-protector\ --param=ssp-buffer-size=4\ -Wformat\ -Wformat-security\ -fmessage-length=0\ -march=armv7-a\ -mfloat-abi=hard\ -mfpu=neon\ -mthumb\ -Wno-psabi QMAKE_CFLAGS_DEBUG=-O2\ -g\ -pipe\ -Wall\ -Wp,-D_FORTIFY_SOURCE=2\ -fexceptions\ -fstack-protector\ --param=ssp-buffer-size=4\ -Wformat\ -Wformat-security\ -fmessage-length=0\ -march=armv7-a\ -mfloat-abi=hard\ -mfpu=neon\ -mthumb\ -Wno-psabi QMAKE_CXXFLAGS_RELEASE=-O2\ -g\ -pipe\ -Wall\ -Wp,-D_FORTIFY_SOURCE=2\ -fexceptions\ -fstack-protector\ --param=ssp-buffer-size=4\ -Wformat\ -Wformat-security\ -fmessage-length=0\ -march=armv7-a\ -mfloat-abi=hard\ -mfpu=neon\ -mthumb\ -Wno-psabi QMAKE_CXXFLAGS_DEBUG=-O2\ -g\ -pipe\ -Wall\ -Wp,-D_FORTIFY_SOURCE=2\ -fexceptions\ -fstack-protector\ --param=ssp-buffer-size=4\ -Wformat\ -Wformat-security\ -fmessage-length=0\ -march=armv7-a\ -mfloat-abi=hard\ -mfpu=neon\ -mthumb\ -Wno-psabi QMAKE_STRIP=: PREFIX=/usr -spec linux-g++ CONFIG+=debug CONFIG+=declarative_debug CONFIG+=qml_debug -o Makefile ) && make -f Makefile 
make[1]: Entering directory `/home/mersdk/share/build-harbour-bodyweight-timer-MerSDK_SailfishOS_armv7hl-Debug/src/insomniac'
make[1]: Nothing to be done for `first'.
make[1]: Leaving directory `/home/mersdk/share/build-harbour-bodyweight-timer-MerSDK_SailfishOS_armv7hl-Debug/src/insomniac'
13:24:17: The process "/home/never/.config/SailfishBeta5/mer-sdk-tools/MerSDK/SailfishOS-armv7hl/make" exited normally.
13:24:17: Deploying binaries...
13:24:17: Starting: "/home/never/.config/SailfishBeta5/mer-sdk-tools/MerSDK/SailfishOS-armv7hl/deploy" --rsync
Building target platforms: armv7hl-meego-linux
Building for target armv7hl-meego-linux
Executing(%install): /bin/sh -e /var/tmp/rpm-tmp.PLGhFe
cd src/insomniac/ && ( test -e Makefile || /usr/lib/qt5/bin/qmake /home/mersdk/share/harbour-bodyweight-timer/src/insomniac/insomniac.pro QMAKE_CFLAGS_RELEASE=-O2\ -g\ -pipe\ -Wall\ -Wp,-D_FORTIFY_SOURCE=2\ -fexceptions\ -fstack-protector\ --param=ssp-buffer-size=4\ -Wformat\ -Wformat-security\ -fmessage-length=0\ -march=armv7-a\ -mfloat-abi=hard\ -mfpu=neon\ -mthumb\ -Wno-psabi QMAKE_CFLAGS_DEBUG=-O2\ -g\ -pipe\ -Wall\ -Wp,-D_FORTIFY_SOURCE=2\ -fexceptions\ -fstack-protector\ --param=ssp-buffer-size=4\ -Wformat\ -Wformat-security\ -fmessage-length=0\ -march=armv7-a\ -mfloat-abi=hard\ -mfpu=neon\ -mthumb\ -Wno-psabi QMAKE_CXXFLAGS_RELEASE=-O2\ -g\ -pipe\ -Wall\ -Wp,-D_FORTIFY_SOURCE=2\ -fexceptions\ -fstack-protector\ --param=ssp-buffer-size=4\ -Wformat\ -Wformat-security\ -fmessage-length=0\ -march=armv7-a\ -mfloat-abi=hard\ -mfpu=neon\ -mthumb\ -Wno-psabi QMAKE_CXXFLAGS_DEBUG=-O2\ -g\ -pipe\ -Wall\ -Wp,-D_FORTIFY_SOURCE=2\ -fexceptions\ -fstack-protector\ --param=ssp-buffer-size=4\ -Wformat\ -Wformat-security\ -fmessage-length=0\ -march=armv7-a\ -mfloat-abi=hard\ -mfpu=neon\ -mthumb\ -Wno-psabi QMAKE_STRIP=: PREFIX=/usr -spec linux-g++ CONFIG+=debug CONFIG+=declarative_debug CONFIG+=qml_debug -o Makefile ) && make -f Makefile install
make[1]: Entering directory `/home/mersdk/share/build-harbour-bodyweight-timer-MerSDK_SailfishOS_armv7hl-Debug/src/insomniac'
install -m 755 -p "libinsomniac.so" "/home/deploy/installroot/usr/share/harbour-bodyweight-timer/lib/harbour/bodyweight-timer/insomniac/libinsomniac.so"
install -m 644 -p /home/mersdk/share/harbour-bodyweight-timer/src/insomniac/qmldir /home/deploy/installroot/usr/share/harbour-bodyweight-timer/lib/harbour/bodyweight-timer/insomniac/
make[1]: Leaving directory `/home/mersdk/share/build-harbour-bodyweight-timer-MerSDK_SailfishOS_armv7hl-Debug/src/insomniac'
lupdate -noobsolete "/home/mersdk/share/harbour-bodyweight-timer/src" "/home/mersdk/share/harbour-bodyweight-timer/qml" -ts "/home/mersdk/share/harbour-bodyweight-timer/translations/harbour-bodyweight-timer.ts" "/home/mersdk/share/harbour-bodyweight-timer/translations/harbour-bodyweight-timer-de.ts" && mkdir -p translations && [ "/home/mersdk/share/build-harbour-bodyweight-timer-MerSDK_SailfishOS_armv7hl-Debug" != "/home/mersdk/share/harbour-bodyweight-timer" -a 1 -eq 1 ] && cp -af "/home/mersdk/share/harbour-bodyweight-timer/translations/harbour-bodyweight-timer-de.ts" "/home/mersdk/share/build-harbour-bodyweight-timer-MerSDK_SailfishOS_armv7hl-Debug/translations" || : ; [ 1 -eq 1 ] && lrelease -nounfinished "/home/mersdk/share/build-harbour-bodyweight-timer-MerSDK_SailfishOS_armv7hl-Debug/translations/harbour-bodyweight-timer-de.ts" || :
Scanning directory '/home/mersdk/share/harbour-bodyweight-timer/src'...
Scanning directory '/home/mersdk/share/harbour-bodyweight-timer/qml'...
Updating '../harbour-bodyweight-timer/translations/harbour-bodyweight-timer.ts'...
    Found 0 source text(s) (0 new and 0 already existing)
Updating '../harbour-bodyweight-timer/translations/harbour-bodyweight-timer-de.ts'...
    Found 0 source text(s) (0 new and 0 already existing)
Updating '/home/mersdk/share/build-harbour-bodyweight-timer-MerSDK_SailfishOS_armv7hl-Debug/translations/harbour-bodyweight-timer-de.qm'...
    Generated 0 translation(s) (0 finished and 0 unfinished)
install -m 644 -p /home/mersdk/share/build-harbour-bodyweight-timer-MerSDK_SailfishOS_armv7hl-Debug/translations/harbour-bodyweight-timer-de.qm /home/deploy/installroot/usr/share/harbour-bodyweight-timer/translations/
cp -f -R /home/mersdk/share/harbour-bodyweight-timer/qml /home/deploy/installroot/usr/share/harbour-bodyweight-timer/
install -m 644 -p /home/mersdk/share/harbour-bodyweight-timer/harbour-bodyweight-timer.desktop /home/deploy/installroot/usr/share/applications/
install -m 644 -p /home/mersdk/share/harbour-bodyweight-timer/harbour-bodyweight-timer.png /home/deploy/installroot/usr/share/icons/hicolor/86x86/apps/
extracting debug info from /home/deploy/installroot/usr/share/harbour-bodyweight-timer/lib/harbour/bodyweight-timer/insomniac/libinsomniac.so
26 blocks
Skip processing files
sending incremental file list
./
usr/
usr/lib/
usr/lib/debug/
usr/lib/debug/.build-id/
usr/lib/debug/.build-id/f3/
usr/lib/debug/usr/
usr/lib/debug/usr/share/
usr/lib/debug/usr/share/harbour-bodyweight-timer/
usr/lib/debug/usr/share/harbour-bodyweight-timer/lib/
usr/lib/debug/usr/share/harbour-bodyweight-timer/lib/harbour/
usr/lib/debug/usr/share/harbour-bodyweight-timer/lib/harbour/bodyweight-timer/
usr/lib/debug/usr/share/harbour-bodyweight-timer/lib/harbour/bodyweight-timer/insomniac/
usr/lib/debug/usr/share/harbour-bodyweight-timer/lib/harbour/bodyweight-timer/insomniac/libinsomniac.so.debug
usr/share/
usr/share/applications/
usr/share/applications/harbour-bodyweight-timer.desktop
usr/share/harbour-bodyweight-timer/
usr/share/harbour-bodyweight-timer/lib/
usr/share/harbour-bodyweight-timer/lib/harbour/
usr/share/harbour-bodyweight-timer/lib/harbour/bodyweight-timer/
usr/share/harbour-bodyweight-timer/lib/harbour/bodyweight-timer/insomniac/
usr/share/harbour-bodyweight-timer/lib/harbour/bodyweight-timer/insomniac/libinsomniac.so
usr/share/harbour-bodyweight-timer/qml/
usr/share/harbour-bodyweight-timer/qml/harbour-bodyweight-timer.qml
usr/share/harbour-bodyweight-timer/qml/components/
usr/share/harbour-bodyweight-timer/qml/components/TimePickerMinutesSeconds.qml
usr/share/harbour-bodyweight-timer/qml/cover/
usr/share/harbour-bodyweight-timer/qml/cover/CoverPage.qml
usr/share/harbour-bodyweight-timer/qml/cover/cover.png
usr/share/harbour-bodyweight-timer/qml/js/
usr/share/harbour-bodyweight-timer/qml/js/database.js
usr/share/harbour-bodyweight-timer/qml/js/global_functions.js
usr/share/harbour-bodyweight-timer/qml/pages/
usr/share/harbour-bodyweight-timer/qml/pages/About.qml
usr/share/harbour-bodyweight-timer/qml/pages/AppSettings.qml
usr/share/harbour-bodyweight-timer/qml/pages/CircleInterval.qml
usr/share/harbour-bodyweight-timer/qml/pages/ExerciseSettings.qml
usr/share/harbour-bodyweight-timer/qml/pages/Home.qml
usr/share/harbour-bodyweight-timer/qml/pages/IntervalSet.qml
usr/share/harbour-bodyweight-timer/qml/pages/Ladder.qml
usr/share/harbour-bodyweight-timer/qml/pages/SuperSet.qml
usr/share/harbour-bodyweight-timer/qml/pages/Tabata.qml
usr/share/harbour-bodyweight-timer/qml/pages/TimerPickerDialogMinutesSeconds.qml
usr/share/harbour-bodyweight-timer/qml/pages/sound/
usr/share/harbour-bodyweight-timer/qml/pages/sound/double_boxing-bell.wav
usr/share/harbour-bodyweight-timer/qml/pages/sound/single_boxing-bell.wav
usr/share/harbour-bodyweight-timer/qml/pages/sound/tripple_boxing-bell.wav
usr/share/harbour-bodyweight-timer/translations/
usr/share/harbour-bodyweight-timer/translations/harbour-bodyweight-timer-de.qm
usr/share/icons/
usr/share/icons/hicolor/
usr/share/icons/hicolor/86x86/
usr/share/icons/hicolor/86x86/apps/
usr/src/
usr/src/debug/
usr/src/debug/src/
usr/src/debug/src/insomniac/

sent 13,886 bytes  received 15,602 bytes  19,658.67 bytes/sec
total size is 1,880,174  speedup is 63.76
13:24:21: The process "/home/never/.config/SailfishBeta5/mer-sdk-tools/MerSDK/SailfishOS-armv7hl/deploy" exited normally.
13:24:21: Elapsed time: 00:08.
```

**Comment by neversun** [comment]
[thanghus](https://github.com/tanghus) helped me out with commit https://github.com/neversun/bodyweight-timer/commit/e68e741649dfcb71cebce95556c5a68c2b2af68c. Project deploys and runs again after that commit.

**Comment by neversun** [comment]
Merged into master c28491bae9f4afbe49d4eab2bdd87277fca456a0
